go/parquet/pqarrow/column_readers.go

```diff
@@ -216,12 +217,34 @@ func (sr *structReader) GetRepLevels() ([]int16, error) {
 }
 
 func (sr *structReader) LoadBatch(nrecords int64) error {
-	for _, rdr := range sr.children {
-		if err := rdr.LoadBatch(nrecords); err != nil {
-			return err
+	var (
+		// REP -- Load batches in parallel
```

Comment:
what is `REP`?

Reply:
Sorry, it's my initials.  I use it to mark places I'm working.

Reply:
That's what I figured. Given this codebase, I'd leave the initials out please. But the comment is otherwise good. 😄